dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1746,3 +1746,12 @@ def pass_time_for_user(name, amount_of_time)
 And(/^one year passes for user "([^"]*)"$/) do |name|
   pass_time_for_user name, 1.year.ago
 end
+
+Then /^I click selector "([^"]*)" (\d+(?:\.\d*)?) times?$/ do |selector, times|
+  step_list = []
+  times.to_i.times do
+    step_list.push("Then I click selector \"#{selector}\" once I see it")
+    step_list.push("And I wait for 1 seconds")
```

Comment:
i just realized that we can actually decrease this sleep to 0.5s (or 0.75s) because the training screen is set to have fish moving at 0.5s ([link](https://github.com/code-dot-org/ml-activities/blob/77531b8e60a120bdc9719185dc82d10fcee1f6ad/src/demo/renderer.js#L83) - `constants.defaultMoveTime` is 1s)

Reply:
I'll leave it as is for now, not sure if it would become more flaky if the framerate drops and there's not enough margin for error.